### DISCUSSION TITLE
Release To Get golang/tools@0500fd4 Fix Out (Already On Master)

### DISCUSSION THREAD

**Discussion by wonderstim** [golangci/golangci-lint#3850] [Q&A] [answered]
Issue with analyzing code that contains goent generated code:
https://github.com/golang/go/issues/58633

Root cause fixed here:
https://github.com/golang/tools/commit/0500fd459260095251c7d8bffc6dcf5c79bfff5c

Dependabot has already updated this repo here:
https://github.com/golangci/golangci-lint/pull/3835

... it just needs a release. 

As a runtime panic during analysis this has no workaround and prevents us from using GolangCI-Lint at all.

**Comment by wonderstim** [comment]
I have confirmed by compiling from source and executing that this fix does in-fact work for preventing the panic during analysis.

**Reply by ldez** [reply, marked as answer]
The v1.53 has been released https://github.com/golangci/golangci-lint/releases
